scripts/ci/ci_before_install.sh

```diff
@@ -15,25 +15,14 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
-set -xeuo pipefail
+set -x
 
 MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-# shellcheck source=scripts/ci/_utils.sh
-. "${MY_DIR}/_utils.sh"
-
-basic_sanity_checks
-
-script_start
+# shellcheck source=scripts/ci/_script_init.sh
+. "${MY_DIR}/_script_init.sh"
```

Comment:
What do you think of changing the pattern for how we import this new file to:

```suggestion
. "$( dirname "${BASH_SOURCE[0]}" )/_script_init.sh"
```

and then also setting `MY_DIR` in it?

Reply:
Why not. Let me try :)

Reply:
Done